libtenzir/builtins/operators/where.cpp

```diff
@@ -6,6 +6,8 @@
 // SPDX-FileCopyrightText: (c) 2021 The Tenzir Contributors
 // SPDX-License-Identifier: BSD-3-Clause
 
+#include "tenzir/select_optimization.hpp"
```

Comment:
Please include with angle brackets for external library includes (which libtenzir is here, as `where` is a plugin).